### ISSUE TITLE
Adding Params to bbob4/cbob4 in AnimeIVTC

### ISSUE THREAD

**Issue by simonsezscott** [realfinder/AVS-Stuff#174]
At this point, if you want to use bbob4_params or cbob4_params in AnimeIVTC, you have to word the string like ", [params here]". So, for example, bbob4_params=", Rep0=7,Rep1=7,Rep2=7". I think it might make more sense to have the comma and space concatenated already so the user doesn't have to insert it themselves, or wonder what the issue is when they don't. Thanks!

**Comment by realfinder** [comment]
I dont think we can do it in another way in avisynth, maybe it can be done in AVS+ with what you want but that will kill compatibility with avs 2.6
